scripts/CustomMaterials.zs

```diff
@@ -846,6 +891,32 @@ MaterialBuilder(11054, "dibenzocrown")
 .colorAverage()
 .build();
 
+MaterialBuilder(11055, "flue_gas")
+.fluid("gas")
+.fluidTemp(1366)
+.color(0x3e3f40)
+.build();
+
+MaterialBuilder(11056, "smoke_bomb_mix")
+.fluid()
+.color(0xbfbfbf)
+.build();
+
+MaterialBuilder(11057, "exhauststeam")
+.fluid("gas")
+.fluidTemp(523)
+.color(0x7a7a7a)
+.build();
+
+MaterialBuilder(11058, "calcium_carbonate_solution")
```

Comment:
This technically exists in CEu in some existing capacity already - as Dissolved Calcium Solution, though it is treated more like a Calcium Acetate solution. Would these be equivalent for how this is used?